components/accordion-menu/demo/demo.js

```diff
@@ -5,8 +5,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import components from 'coherent-gameface-components';
-import AccordionMenu from '../umd/accordion-menu.development';
+import AccordionMenu from '../script.js';
```

Comment:
Why do we import from the script.js instead of index.js which is the main file of the component?

Reply:
Because I wanted to avoid importing the components library. This way we import only script.js and webpack bundles everything.